src/components/Header/Header.jsx

```diff
@@ -47,19 +47,18 @@ export const Header = (isOpen) => {
             <Navbar />
           </Box>
 
-          <Flex justify="center">
-            <NotificationsMenu />
-            <Box
-              display={{ base: isOpen ? 'none' : 'block', lg: 'block' }}
-              flexBasis={{ base: '100%', md: 'auto' }}
-              ml={{ base: '8px', xl: '10px' }}
-              mr={{ lg: 0, xl: '10px' }}
-            >
-              <UserName />
-            </Box>
+          <NotificationsMenu />
+          <Box
+            display={{ base: isOpen ? 'none' : 'block', lg: 'block' }}
+            flexBasis={{ base: '100%', md: 'auto' }}
+            ml={{ base: '8px', xl: '10px' }}
+            mr={{ lg: 0, xl: '10px' }}
+            justifyContent="center"
```

Comment:
That line does nothing. You can safely remove it 😉

Reply:
done